docs/content/users/usage/commands.md

```diff
@@ -158,6 +158,9 @@ ddev config
 # Configure a Drupal 8 project with a `web` document root
 ddev config --docroot=web --project-type=drupal8
 
+# Automatically detect `docroot` and `project-type` without prompting
```

Comment:
```suggestion
# Automatically detect `docroot` and `project-type` without prompting
# This assumes that the code is already in place in the directory and auto-detection is possible
```

Reply:
@rfay I think it might make more sense to point this out [where the documentation explains what the flag does](https://github.com/ddev/ddev/blob/c51a90500868f487dc21b61726599646d729d368/docs/content/users/usage/commands.md?plain=1#L172), instead of using comments on the example. If it’s important to say this here, maybe tacking on `(if possible)` would suffice?

Reply:
Or “Attempt to automatically detect (...)”

Reply:
That would be OK. I'm reluctant with --auto because if there's no code people get bogus settings that then haunt them. 

Little-known fact: `--auto` doesn't do anything. It just prevents the interactive questions about docroot and type..  So `ddev config --php-version=8.2` is also auto, but it sort-of does something.

Reply:
Sounds like `--sort-of-auto` defaults to `true`. :)

Tricky when the arguments kind of imply 🤫 and DDEV probably should say “hey your docroot setting may be weird because you’ve given me nothing to work with.” I’d personally prefer a warning or refusal on DDEV’s part but I can see how that’d be an unwanted conflict for somebody else. (Or maybe even DDEV internally in some case?)

Reply:
`--auto` was only added after years of recommending some random arg like `--php-version`. People REALLY WANTED IT :) But the thing is those new users can get messed up.